src/test/shared/cloudformation/cloudformation.test.ts

```diff
@@ -136,14 +136,13 @@ describe('CloudFormation', () => {
             assert.deepStrictEqual(loadedTemplate, expectedTemplate)
         })
 
-        it('only loads YAML with valid types', async () => {
-            // timeout is not a number
+        it('only loads YAML with valid fields', async () => {
+            // codeuri is missing
```

Comment:
Timeout is still not a number. This should still fail due to that as well...either make a second test or change that detail.

Reply:
Ah this is because I also changed that but changed it back, forgot to change the test